cloud/aws/services/ec2/ami.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ec2
+
+// DefaultAMILookup returns the default AMI based on region
+func (s *Service) defaultAMILookup(region string) string {
+	//TODO(chuckha) tags on our built images need to be copied to the public images
```

Comment:
I'd update this comment to say that we cannot use tags for lookup here and that we need to determine a better method of filtering for public AMIs.

Reply:
I'll update it, ticket exists here https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/138